Token_Contracts/contracts/HumanStandardToken.sol

```diff
@@ -19,7 +19,7 @@ contract HumanStandardToken is StandardToken {
 
     function () {
```

Comment:
I think one can now actually just take this out, since functions require payable to receive eth: even fallback functions. If it's not there, then it will throw.